### ISSUE TITLE
Deal with Papertrail clippage

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#4171]
Previous: #1688.

>Your Papertrail service for "gratipay" has reached its log data transfer limit of **50 MB** for today.
Based on recent volume, your projected total daily log data transfer is **145.5 MB**.
>
>**How's this work?**
>
>At capacity, new logs are dropped. Existing logs aren't affected. Processing automatically resumes the following day.
>
>Not sure what's best for this app, or have a question or request? Just reply.
>
>**10 minutes of tuning goes a long way**
>
>We're all about effective logs. If this is unexpected, an app may be generating more log data than is useful or intended. This email isn't to nag you to upgrade; we've invested a lot of effort to provide easy, free ways to manage log volume (and as users ourselves, we'd never want log processing to unexpectedly stop). Here's some options:
>
>- **Have Papertrail [filter noise](http://blog.papertrailapp.com/more-signal-less-noise-with-simple-free-log-filtering/)** free, such as requests for static assets.
>- **Denser logging.** Many web frameworks (including Rails) are very verbose. We have tips to create denser, more informative logs (here's a few for [Rails](http://help.papertrailapp.com/kb/configuration/controlling-verbosity)). Just reply and tell us which frameworks or daemons you use.
>- **Punt and tune later.** Papertrail plans average 15,000 messages per $0.01 (that's 1/15,000th of a cent per message).
>- **Ask us.** Just reply.
>
>We're out to provide awesome troubleshooting knowledge and a great value. We don't care whether that's $0 or $300 as long as it's solving your problem well.
>
>Fees are pro-rated to the second. To change plans at any time, visit [heroku.com](https://dashboard.heroku.com/apps/gratipay/resources) or run:
heroku addons:upgrade papertrail:new-plan-name --app gratipay
>
>Happy logging,
the Papertrail team

https://gratipay.freshdesk.com/helpdesk/tickets/6021

**Comment by whit537** [comment]
So I guess this has been going on for a while, but we just started receiving (daily!) emails, because of a newly added email to our [Papertrail notifications](https://papertrailapp.com/account/usage-notifications). The new email is `gratipay@herokumanager.com`, and it was added three days ago. I find [this](https://devcenter.heroku.com/articles/add-on-provider-technical-best-practices#use-the-correct-email-address-for-customers) on the web:

> If you see an email address like `somename@herokumanager.com` then this customer is a Heroku Enterprise customer, and the `@herokumanager.com` address will send email to each of the organization’s admins. You can use this email address like any other email address.

We're not an Enterprise customer (that I know of) but probably the Heroku Teams [switch](https://github.com/gratipay/inside.gratipay.com/issues/871) is what's behind this. (P.S. I've just sent a test message from chad@zetaweb.com to see if this is open for anyone to send mail to.)

@rohitpaulk et al. Should we fix the problem or turn off the notifications?
